Package.swift

```diff
@@ -894,7 +894,10 @@ let package = Package(
       dependencies: [
         "FirebaseCore",
         "FirebaseInstallations",
-        "FirebaseRemoteConfig",
+        // Performance depends on the Obj-C target of FirebaseRemoteConfig to
```

Comment:
Is this an optimization only? Or is there a functional reason?

If functional, is CocoaPods ok?

Reply:
Discussed offline, but in summary:

> Is this an optimization only? Or is there a functional reason?

Yes, an optimization only.

> If functional, is CocoaPods ok?

CocoaPods does not allow for a similar optimization as there is no separation between the Objective-C and Swift sources.